.examples/nginx/docker-compose.yml

```diff
@@ -1,26 +1,40 @@
-db:
-  image: mariadb:latest
-  volumes:
-    - ./mysql/runtime2:/var/lib/mysql
-  environment:
-    - MYSQL_ROOT_PASSWORD
-app:
-  image: matomo:fpm
-  links:
-    - db
-  volumes:
-    - ./config:/var/www/html/config:rw
-    - /home/your-logs-folder:/var/www/html/logs
-  env_file:
-    - ./matomo.env
-web:
-  image: nginx:latest
-  volumes:
-    - ./nginx.conf:/etc/nginx/nginx.conf:ro
-  links:
-    - app
-  volumes_from:
-    - app
-  environment:
-    - VIRTUAL_HOST
+version: '3'
 
+services:
+  db:
+    image: mariadb
+    command: --max-allowed-packet=64MB
```

Comment:
Is it possible that you make a quick README.md to explain `command: --max-allowed-packet=64MB` and also from where you inspired yourself for the nginx conf ?

Reply:
The Matomo system check recommends to set `max_allowed_packet=64MB`